addons/l10n_in_edi/views/edi_pdf_report.xml

```diff
@@ -17,14 +17,14 @@
     <template id="external_layout_boxed_inherit_l10n_in_edi" inherit_id="web.external_layout_boxed">
         <img t-if="company.logo" t-att-src="image_data_uri(company.logo)" position="after">
             <div class="mt-5" t-if="l10n_in_einvoice_json">
-                <span style="font-size: 10px;">IRN:<span t-esc="l10n_in_einvoice_json['Irn']"/></span>
+                <span style="font-size: 10px;"><strong>IRN:</strong><span t-esc="l10n_in_einvoice_json['Irn']"/></span>
             </div>
         </img>
         <div class="col-6 text-right mb4" position="before">
             <div class="col-3" t-if="l10n_in_einvoice_json">
                 <div class="text-center">
                     <img t-att-src="'/report/barcode/?type=%s&amp;value=%s&amp;width=%s&amp;height=%s' %
-                    ('QR', l10n_in_einvoice_json['SignedQRCode'], 150, 150)"/>
+                    ('QR', l10n_in_einvoice_json['SignedQRCode'], 1000, 1000)" style="max-height: 180px"/>
```

Comment:
that big?  And then crop?

Reply:
using 150 we get a blurry QR code so we set 1000 and after the max height to 180px
we can still reduce from 500 to 300 but not lower than 300.